lib/fathead/npm_cli/parse.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import requests
+import re
+from bs4 import BeautifulSoup
+
+
+START_URL = "https://docs.npmjs.com/cli/access"
```

Comment:
This typically isn't practiced in a FH. People retrieve the raw document in *fetch.sh* and parse from there. I'm fine with this, but maybe others won't be. Let's wait and see